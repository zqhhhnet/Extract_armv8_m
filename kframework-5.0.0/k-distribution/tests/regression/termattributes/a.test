//Test to ensure attributes propagate through parser, kompile, and java backend
(V:Int{foo} ~> foo::A{bar} ~> V ~> V:Int{foo})::K{baz}
