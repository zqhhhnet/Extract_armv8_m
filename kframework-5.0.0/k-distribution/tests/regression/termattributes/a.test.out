V:Int{ foo()} ~> foo ::A{ bar()} ~> V ~> V ::K{ baz()}
